src/components/header/Navigation.tsx

```diff
@@ -13,12 +15,12 @@ function Navigation({ isOpen, page }: NavigationProps): React.ReactElement {
     <nav>
       <Styled.NavList isOpen={isOpen}>
         <Styled.ListItem isActive={page === "index"}>
-          <Link href="/">
+          <Link href="/" passHref>
```

Comment:
[The `passHref` is only needed when passing to a styled-component](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag)